### ISSUE TITLE
Darknet reference

### ISSUE THREAD

**Issue by nsantavas** [AlexeyAB/darknet#966]
Hi 
I am trying to train Darknet Reference model without pretrained weights to use it as detector for my own single object, but I have these results. What is going on? How many filters should I create in cfg file?

![screenshot from 2018-06-02 16-31-02](https://user-images.githubusercontent.com/37878833/40876795-955b5d0c-6682-11e8-9e20-7218cbc1ca6e.png)

```
[net]
# Train
 batch=128
 subdivisions=4
# Test
#batch=1
#subdivisions=1
height=256
width=256
channels=3
momentum=0.9
decay=0.0005
max_crop=320

learning_rate=0.1
policy=poly
power=4
max_batches=1600000

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[convolutional]
filters=30
size=1
stride=1
pad=1
activation=leaky

[avgpool]

[softmax]
groups=1

```

**Comment by AlexeyAB** [comment]
@nsantavas Hi,
Darknet Reference model isn't Detector as Yolo. 
Darknet Reference model `darknet.cfg` is a Classifier - it can predict only what objects are on the image, but not where are they: https://github.com/AlexeyAB/darknet/blob/4cf4a89f05c671041c217a724fc9b4b026eb9de9/build/darknet/x64/classifier_resnet50.cmd#L1

More: https://pjreddie.com/darknet/imagenet/

Read these issues:
* https://github.com/AlexeyAB/darknet/issues/876
* https://github.com/AlexeyAB/darknet/issues/928
* https://github.com/AlexeyAB/darknet/issues/679

**Comment by nsantavas** [comment]
Thank you. Is there any model lighter than tiny yolo v3 that you suggest for real time computer vision on embedded system?

**Comment by AlexeyAB** [comment]
There is SSD MobileNet on Caffe-framework that is much more lighter but has much less accuracy.
Or you can just decrease network resolution (width= height= in cfg) of tiny yolo v3 to increase speed.
